.travis.yml

```diff
@@ -0,0 +1,22 @@
+ # Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think Travis-CI is great and use it all the time, but I think perhaps there should be some discussion on-list about it first. There is already some CI somewhere (although I don't where and I'm not sure that ordinary mortals have access to it-- @afs ?).